### PR TITLE
Bugfix/resources title adjustments 97

### PR DESCRIPTION
**Pull request by pk-hotovo**
issue 97

### CODE REVIEW

File: src/styles/main.css [hunk 1]
```diff
@@ -114,7 +114,10 @@
 }
 
 .swagger-ui section.models.is-open h4 {
-  border-bottom: none;
+  border-bottom: 1px solid var(--black-200);
+}
+.swagger-ui section.models.is-open h4:hover {
+  border-bottom: 1px solid var(--green-500);
```
Comment (tomaszwylezek) [inline src/styles/main.css]: It shouldn't change color to green

File: src/styles/main.css [hunk 2]
```diff
@@ -114,7 +114,10 @@
 }
 
 .swagger-ui section.models.is-open h4 {
-  border-bottom: none;
+  border-bottom: 1px solid var(--black-200);
```
Comment (tomaszwylezek) [inline src/styles/main.css]: border-bottom should be `--section_colors-stroke` 
Same for the whole border of Shemas/Models

**Review by tomaszwylezek — CHANGES_REQUESTED** [review]
**Expected result**
Screenshot 2022-07-21 at 09 23 35

**Previous result**
Screenshot 2022-07-21 at 09 24 44

**Current branch result**
Screenshot 2022-07-21 at 09 24 10

The changes wouldn't get rid of margin on both side of the container, that change is unexpected.
The titles like `auth` `HTTP methods and `Status code` are still not aligned with boxes

**Comment by pk-hotovo** [comment]
The titles like auth HTTP methods and Status code` are still not aligned with boxes - can you please explain what it means in this context?

**Comment by pk-hotovo** [comment]
updated

File: src/styles/main.css [hunk 3]
```diff
@@ -226,7 +229,7 @@
   word-break: break-word !important;
   border: 1px solid var(--section_colors-stroke);
   margin: var(--spacing-xl) 0;
-  padding: var(--spacing-xl);
+  padding: var(--spacing-xl) 0;
```
Comment (tomaszwylezek) [inline src/styles/main.css]: it shouldn't change

**Review by tomaszwylezek — CHANGES_REQUESTED** [review]
It is not aligned with the boxes
Screenshot 2022-07-21 at 14 45 30
We need to remove that padding-left from that button

**Review by tomaszwylezek — APPROVED** [review]
lgtm

**Pull request merged**